src/Car.java

```diff
@@ -0,0 +1,14 @@
+public class Car {
+
+    public void start() {
+        startElectricity();
+        startCommand();
+        startFuelSystem();
+    }
+
+    private void startElectricity() {}
+
```

Comment:
Please add implementation to the interbal methods

Reply:
Done